docs/intro.md

```diff
@@ -0,0 +1,94 @@
+# Sashimi
+
+<a href="url"><img 
+src="https://github.com/portugueslab/sashimi/blob/master/sashimi/icons/main_icon.png"
+align="left" 
+height="190" 
+width="270"></a>
+
+![tests](https://github.com/portugueslab/sashimi/workflows/tests/badge.svg?branch=master)
+[![Docs](https://img.shields.io/badge/docs-dev-brightgreen)](https://portugueslab.github.io/sashimi/)
+[![Coverage Status](https://coveralls.io/repos/github/portugueslab/sashimi/badge.svg)](https://coveralls.io/github/portugueslab/sashimi)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4122062.svg)](https://doi.org/10.5281/zenodo.4122062)
+[![Python 3.8](https://img.shields.io/badge/python-3.8-blue.svg)](https://www.python.org/)
+
+Sashimi is a user-friendly software for efficient control of digital scanned light sheet microscopes (DSLMs).
+It is developed by members of the [PortuguesLab](http://www.portugueslab.com/)
+ at the Technical University of Munich and Max Planck Institute of Neurobiology. Sashimi relies on the fast, multidimensional
+ [Napari viewer](https://github.com/napari/napari).
+ 
+While built for a particular microscope configuration, the modular architecture allows for easy replacement of
+hardware by other vendors (we will help with and welcome contributions for supporting other cameras, boards and light sources).
+ 
+
+ 
+## Installation
+
+[Install the latest Anaconda](https://www.anaconda.com/) distribution of Python 3.
+
+Clone this repository and navigate to the main folder `../sashimi`
+
+### Recommended: Create a new environment
+
+It is a good practice to create an environment for every project. The provided `environment.yml` sets up almost all required dependencies (see below).
+
+    conda env create -f {path to environment.yml}
+
+You can activate the environment by running:
+
+    conda activate sashimi
+    
+After this you moght have to install two extra dependencies for controlling a Cobolt laser:
```

Comment:
Technically this should read for controlling a laser through serial communication, such as Cobolt lasers.